tests/queries/0_stateless/00700_decimal_formats.sql

```diff
@@ -9,22 +9,22 @@ CREATE TABLE IF NOT EXISTS decimal
 
 INSERT INTO decimal (a, b, c) VALUES (42.0, -42.0, 42) (0.42, -0.42, .42) (42.42, -42.42, 42.42);
 INSERT INTO decimal (a, b, c) FORMAT JSONEachRow {"a":1.1, "b":-1.1, "c":1.1} {"a":1.0, "b":-1.0, "c":1} {"a":0.1, "b":-0.1, "c":.1};
+
 INSERT INTO decimal (a, b, c) FORMAT CSV 2.0,-2.0,2
-;
+
```

Comment:
The old implementation used \n as the separator, but now, we uses \n\n.  The extra semicolons will be parsed as part of insert_data, which will affect the correctness of the data.

Reply:
One option is to also allow
`\n;` in addition to `\n\n`

Reply:
A new option is overkill, IMHO. Allowing both `\n\n` and `\n;` as separators by default would be ideal.

Reply:
The situation is complicated. There are not many people using `\n`;. I think it is better not to support it directly, otherwise there are other formats that need to support different situations.